modules/lges/components/LgeListItem.tsx

```diff
@@ -0,0 +1,147 @@
+import { Box, Grid, GridItem, HStack, Link, Text } from '@chakra-ui/react';
+import { BoxProps } from '@chakra-ui/layout';
+import { useGetLgeToken } from './lib/useGetLgeToken';
+import TokenAvatar from '~/components/token/TokenAvatar';
+import { formatDistanceToNow } from 'date-fns';
+import { GqlLgeExtended } from '~/modules/lges/useLgeList';
+import { IconTwitter } from '~/components/icons/IconTwitter';
+import { IconDiscord } from '~/components/icons/IconDiscord';
+import { IconTelegram } from '~/components/icons/IconTelegram';
+import { IconMedium } from '~/components/icons/IconMedium';
+import { IconGlobe } from '~/components/icons/IconGlobe';
+import React, { ReactNode } from 'react';
+
+interface Props extends BoxProps {
+    lge: GqlLgeExtended;
+}
+
+function getStatusText(lge: GqlLgeExtended) {
+    switch (lge.status) {
+        case 'active':
+            return `Ends in ${formatDistanceToNow(new Date(lge.endDate))}`;
+        case 'upcoming':
+            return `Starts in ${formatDistanceToNow(new Date(lge.startDate))}`;
+        case 'ended':
+            return `Ended ${formatDistanceToNow(new Date(lge.endDate))} ago`;
+    }
+}
+
+function getIconLink(urlType: keyof typeof lge, lge: GqlLgeExtended): ReactNode {
+    const href = lge[urlType] as string;
+
+    if (!href) {
+        return null;
+    }
+
+    const props = {
+        color: '#c1c1d1',
+    };
+
+    const hoverProps = { color: 'beets.highlight' };
+
+    let iconType;
+    switch (urlType) {
+        case 'discordUrl':
+            iconType = <IconDiscord {...props} _hover={hoverProps} />;
+            break;
+        case 'mediumUrl':
+            iconType = <IconMedium {...props} _hover={hoverProps} />;
+            break;
+        case 'telegramUrl':
+            iconType = <IconTelegram {...props} _hover={hoverProps} />;
+            break;
+        case 'twitterUrl':
+            iconType = <IconTwitter {...props} _hover={hoverProps} />;
+            break;
+        case 'websiteUrl':
+            iconType = <IconGlobe {...props} _hover={hoverProps} />;
+            break;
+    }
+    return (
+        <Link href={href} target="_blank" color="gray.100">
+            {iconType}
```

Comment:
it must be possible to do something like `<IconType {...props} />` where `props` also contains the `_hover` prop?

Reply:
Should be, but it may bite you in the ass. Overrriding `_hover` makes it that the previously applied hover styles are wiped.